src/app/components/TxQueue/TxQueue.tsx

```diff
@@ -7,11 +7,14 @@ import { TxQueueCard } from './TxQueueCard';
 
 export const TxQueue = ({ queue }: { queue: TxQueueEntry[] }) => (
   <>
-    {queue.map((q) => (
-      <React.Fragment key={q.uuid}>
-        <TxQueueCard item={q} />
-        <Divider />
-      </React.Fragment>
-    ))}
+    {queue
+      .slice()
+      .sort((a, b) => b.timestamp - a.timestamp)
```

Comment:
Can we move this logic to the component that is passing the `queue` prop? Maybe a Redux selector?